AdyenActions/Components/Await/PollingComponent.swift

```diff
@@ -12,6 +12,12 @@ import Foundation
 /// A specific await component thats keeps polling the `/status` endpoint to check the payment status.
 internal final class PollingComponent: AnyPollingHandler {
     
+    private let maxErrorNumber = 3
```

Comment:
3 is too much, 1 is enough

Reply:
Right now it fires 2 times max.
User might potentially go back and forth to external app several times + I see no harm in retrying 1 extra time in case of network error 🤔